### ISSUE TITLE
ammo looting should be restrictable by damage

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#127]
Nexus suggestion

> on the oldrim version of AHSE, the mod author briefly had Arrow looting set up instead of V/W, it had arrow damage values. So you could say set it up to not loot any arrows less than 10 damage or 12 damage, or whatever. The last version of AHSE and SHSE, has that option blank now.
I like looting 10 damage arrows at the start of the game, but as I level up and start getting better arrows, I move it up to 12 or 14 and increase it as I get more better arrows. Since he removed the feature, I've always got around it by just setting it to never loot arrows, but adding and removing them from the whitelist manually.

**Comment by SteveTownsend** [comment]
The worth of arrows is already calculated using damage. Just need to add a different slider for them.

**Comment by SteveTownsend** [comment]
Test evidence with new logic: too little damage
```
J:\GitHub\SmartHarvestSE\src\Data\dataCase.cpp(890): [VERBOSE] {14396} (2020-07-15 03:56:20 PM) OK, not in flight, change in arrow position dx=-36.41,dy=-46.79,dz=-34.34
J:\GitHub\SmartHarvestSE\src\FormHelpers\FormHelper.cpp(121): [VERBOSE] {14396} (2020-07-15 03:56:20 PM) Ammo Chitin Arrow(410139f7) damage = 7.00
J:\GitHub\SmartHarvestSE\src\FormHelpers\IHasValueWeight.cpp(50): [VERBOSE] {14396} (2020-07-15 03:56:20 PM) Checking value: 7
J:\GitHub\SmartHarvestSE\src\FormHelpers\IHasValueWeight.cpp(70): [VERBOSE] {14396} (2020-07-15 03:56:20 PM) Chitin Arrow/0x410139f7 ammo damage 7 vs threshold 12.00
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(438): [VERBOSE] {14396} (2020-07-15 03:56:20 PM) block - v/w excludes harvest for 0x410139f7
```
Enough damage:
```
J:\GitHub\SmartHarvestSE\src\Data\dataCase.cpp(890): [VERBOSE] {14396} (2020-07-15 03:56:20 PM) OK, not in flight, change in arrow position dx=-19.86,dy=-50.36,dz=-12.86
J:\GitHub\SmartHarvestSE\src\FormHelpers\FormHelper.cpp(121): [VERBOSE] {14396} (2020-07-15 03:56:20 PM) Ammo Daedric Arrow(000139c0) damage = 24.00
J:\GitHub\SmartHarvestSE\src\FormHelpers\IHasValueWeight.cpp(50): [VERBOSE] {14396} (2020-07-15 03:56:20 PM) Checking value: 24
J:\GitHub\SmartHarvestSE\src\FormHelpers\IHasValueWeight.cpp(70): [VERBOSE] {14396} (2020-07-15 03:56:20 PM) Daedric Arrow/0x000139c0 ammo damage 24 vs threshold 12.00
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(442): [VERBOSE] {14396} (2020-07-15 03:56:20 PM) Daedric Arrow/0x000139c0 value:24
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(471): [VERBOSE] {14396} (2020-07-15 03:56:20 PM) SmartHarvest Daedric Arrow/0x000139c0 for REFR 0xff00230b
```

**Comment by SteveTownsend** [comment]
Fix ready for v1.8.0.0, MCM looks and works OK
